pmm-app/tests/pages/dashboardPage.js

```diff
@@ -320,6 +388,7 @@ module.exports = {
       "//span[contains(text(),'No Data')]//ancestor::div[contains(@class,'panel-container')]//span[contains(@class,'panel-title-text')]",
     collapsedDashboardRow: "//div[@class='dashboard-row dashboard-row--collapsed']/a",
     annotationMarker: "(//div[contains(@class,'events_marker')])",
+    clearSelection: "//a[@ng-click='vm.clearSelections()']",
```

Comment:
nit: It doesn't seem like a good selector, since it's an angular stuff which grafana is migrating away from.

Reply:
I mean, most likely it'll break quite soon )

Reply:
yes, will update it, couldn't find best alternative, will still check